### ISSUE TITLE
Filter out senior tech positions

### ISSUE THREAD

**Issue by boushrabettir** [acmcsufoss/acmOpportunities#3]
Some of the job postings have senior positions, which doesn't make sense in our case since most of everyone is trying to look for a entry level position.

Skip over position titles containing any type of abbreviations such as `Sr` or `Senior` etc.

**Comment by boushrabettir** [comment]
- Note: Any criticism about this could be helpful. I only added two lines of code so, I am definitely open to suggestions to anyone who looks at this.

Looked over how senior positions are written down per say, and it seems the only way that its written down is `Senior` and `Sr`. Tested out a simple conditional on LinkedIn jobs and here was the result. (Made sure also to comment everything out so I don't waste requests)
![image](https://github.com/acmcsufoss/acmOpportunities/assets/116927138/1090cfc5-7f9d-4162-bce9-42a5f62ab483)


Tested it out also in the `list_all_opportunities()`. I deleted the conditional in the function, though, as it was only used for testing purposes. Here is a sample of what I received. I think it's able to filter it out well.
![image](https://github.com/acmcsufoss/acmOpportunities/assets/116927138/62234498-7af4-44ee-bf6a-f4ae20887044)
![image](https://github.com/acmcsufoss/acmOpportunities/assets/116927138/cef335f4-5d54-4a31-81b5-facfd4c0af21)
